finality-aleph/src/network/manager/service.rs

```diff
@@ -242,7 +248,7 @@ impl<NI: NetworkIdentity, D: Data> Service<NI, D> {
             None => {
                 let (data, data_from_network) =
                     self.start_validator_session(pre_session, addresses).await?;
-                return Ok((None, data, data_from_network));
+                return Ok(((None, data), data_from_network));
```

Comment:
naming is hard :sweat_smile: but at the same time it's kinda hard to follow what is going on here with the tuples  usd for these complex types - could we use structs (with named fields) to untangle it a bit?